build_scripts/build_linux_deb-1-gui.sh

```diff
@@ -2,17 +2,13 @@
 
 set -o errexit
 
-echo "Installing global npm packages"
-cd npm_linux || exit 1
-npm ci
-
-cd ../../ || exit 1
```

Comment:
## For reviews
These lines were required only for `npx lerna clean -y`  below but actually `npx lerna ...` invokes `lerna` installed in `chia-blockchain-gui/node_modules`.
We can safely remove npm installation here without any breaking changes.